docs/src/route.ts

```diff
@@ -1,5 +1,10 @@
 const ROUTES = {
-  toolpadDocs: '/toolpad/getting-started/setup',
+  TOOLPAD_DOCS: '/toolpad/getting-started/setup',
```

Comment:
Why not match the convention in https://github.com/mui/material-ui/blob/1c9b514738037e81fcbc1b434191cab128eb64ed/docs/src/route.ts#L3?

```suggestion
  toolpadDocs: '/toolpad/getting-started/setup',
```